### DISCUSSION TITLE
More Games on the Platform?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [modrinth/.github#133] [Other discussions]
Do you intend to open the platform to other games or is the focus only on Minecraft?

Do you have an estimated date of when you guys will add support for new games?

**Comment by falseresync** [comment]
Citation from Discord:

> **Modrinth support for games other than Minecraft**
> Modrinth is planning on supporting games other than Minecraft in the distant future. We'd rather focus on perfecting Minecraft support before we work on other games, though!


>**Regarding ETAs for new Modrinth features or fixes**
>Modrinth does not provide any public ETAs for new features or for bug fixes. You can take a look at [our roadmap](https://roadmap.modrinth.com/) to see what features we're working on right now as well as what is yet to come. If you'd like to help us out by making pull requests, please feel free to! All of our code is open source [on GitHub](https://github.com/modrinth).

It's best if you use Discord for general questions, discussions are primarily for concrete feature requests

**Reply by Heus-Sueh** [reply]
thanks
